client/components/board/boardFactory.js

```diff
@@ -3,27 +3,48 @@ angular.module('OpenEXP')
         const OpenBCIBoard = require('openbci-sdk');
         const ourBoard = new OpenBCIBoard.OpenBCIBoard();
 
-        var findPorts = () => {
+        // wrote a method in the openbci-sdk to return an array of ports and
+        // wrapping it in a promise here
+        var listDevices = () => {
```

Comment:
true, we could go directly to serialport for this functionality. but for now, i think it makes sense since its the only device we have.